source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+- Distributing your applications via public or private app stores
+
+Your custom built apps are a fork of the Mattermost project. This means you will be responsible for adding in any feature or security updates.  
+
+To understand what’s involved, have your development team `read through our documentation <https://developers.mattermost.com/contribute/mobile/build-your-own/>`_.
+
+    **Note:** As of Mattermost 5.18, E20 customers can limit data sent to the HPNS. With this option a message containing only an ID. Once the mobile client receives this ID, the message contents are loaded from the server. Thus, the message contents are never transmitted through APNS/FCM.
+
+----
+
+#####################################################
+Mobile Deployment via Public App Stores (Recommended)
+#####################################################
+
+To proceed you must have a Mattermost Server installed and accessible. This is also true for  using the hosted version of the Mattermost push proxy (HPNS).
+
+*********************************
+Accessing the Mobile Applications
+*********************************
+
+The Mattermost mobile application is available for both Android and iOS devices. At this point, it’s as simple as having your users download the application and `point to your Mattermost Server URL <https://docs.mattermost.com/help/getting-started/signing-in.html#ios-setup>`_.
+
+    **Note:** The Mattermost Mobile apps are signed and have certificates associated with Mattermost and the public app stores. This means they will not work if you are privately hosting the Mattermost Push Proxy Service.
+
+`Mattermost for Android Devices <https://play.google.com/store/apps/details?id=com.mattermost.rn&hl=en_US>`_ (via Google Play)
+`Mattermost of iOS Devices <https://apps.apple.com/us/app/mattermost/id1257222717>`_ (via Apple App Store)
+
+If you do not desire (or require) additional security provide via an EMM Provider, your deployment is complete. Feel free to point your users to the `available documentation <https://docs.mattermost.com/guides/user.html>`_.
+
+The sub-sections below serve as a high-level guide to understanding this deployment model. When necessary, we point to the documentation for technical instruction.
+
+    **Note:** At a minimum, we recommend you follow our recommended steps to secure your deployment.
+
+********************************************
+Using an EMM Provider with Public Store Apps
+********************************************
+
+EMM providers help extend security parameters to the Mattermost mobile applications. The AppConfig standard makes this possible. `Review the Mattermost AppConfig Values Documentation <https://docs.mattermost.com/mobile/mobile-appconfig.html#mattermost-appconfig-values>`_ for a complete list of available parameters.  
+
+When going this route, you should consider…
+
+- What is the mobile policy, is it company-owned or BYOD?
+- Do you know what devices will be used if customer-owned?
```

Comment:
customer-owned? what are we trying to say here?

Reply:
BYOD vs. Company-distributed.

Reply:
+1 understand the question better now. Replace we can replace customer-owned with BYOD. Need to rewrite the sentence to make it rad correctly.

Reply:
solved in meeting

Reply:
+1